dvc/command/metrics.py

```diff
@@ -139,6 +141,12 @@ def add_parser(subparsers, parent_parser):
         action='store_true',
         default=False,
         help='Show metrics for all tags.')
+    metrics_show_parser.add_argument(
+        '-R',
+        '--recursive',
+        action='store_true',
+        default=False,
+        help='Show metrics recursively.')
```

Comment:
we should probably update the position argument description - it accepts DVC file or a directory now, right?

Reply:
Can we do something like: 'Recursively search and process metric files in path.'? Or something else, just to make it a little bit more clear what's happening here.